terraform/gitlab.tf

```diff
@@ -24,7 +24,8 @@ resource "kubernetes_config_map" "terraform-gitlab-info" {
     "redishost"                = aws_elasticache_replication_group.gitlab.primary_endpoint_address
     "redisport"                = var.redis_port
     "ingress-security-groups"  = aws_security_group.gitlab-ingress.id
-    "gitlab_name"              = "gitlab.teleport-${var.cluster_name}.${var.domain}"
+    # "gitlab_name"              = "gitlab.teleport-${var.cluster_name}.${var.domain}"
```

Comment:
should this be deleted?

Reply:
Yeah.  Good question.  I will delete this.  Thanks!